README.md

```diff
@@ -1,86 +1,115 @@
+# remoteworks.jp
```

Comment:
I think we don't need `# remoteworks.jp` ~ `## License` sections
if you wanna have manual for development, create separate file like `DEVELOPMENT.md`

Reply:
Thanks for the tip, created `DEVELOPMENT.md` file.